SegmentedControlTab.js

```diff
@@ -34,6 +36,10 @@ const TabOption = ({
             isTabActive ? [styles.activeTabStyle, activeTabStyle] : {},
             firstTabStyle,
             lastTabStyle]}
+            accessible={accessible}
+            accessibilityLabel={accessibilityLabel}
+            accessibilityTraits={isTabActive ? "selected" : "button"}
```

Comment:
`accessibilityTraits` is nice to have 😊`selected makes more values` but `button` is used when the element should be treated as a button. in this case, t's tabs. I'm not sure what we can use for Tabs. If we do not have any if prefer to keep the value as `none` for inactive state.